examples/evaluate/04-TBROM_images.py

```diff
@@ -96,8 +106,9 @@ def snapshot_to_cfd(snapshot_file, geometry_file, field_name, outputFilePath):
 
 
 ###############################################################################
-# Loading the Twin Runtime and generate the temperature results from the TBROM
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Load the twin runtime and generate temperature results
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Load the twin runtime and generate temperature results from the TBROM.
```

Comment:
I've never seen TBROM defined anywhere. What does TBROM stand for?

Reply:
It stands for Twin Builder ROM. Maybe we should better use Twin Builder ROM instead of TBROM anywhere in the doc? @chrpetre @PipKat what do you think?

Reply:
I think we can use TROM in the doc. However, we need to define TROM the first time that it is used. This seems to be here: [examples/evaluate/04-TBROM]. The other occurrences I found in my search of the repository were in tests or in variable definitions in this file: [src/ansys/pytwin/evaluate/twin_model.py](https://github.com/pyansys/pytwin/blob/c82de5864adcb167431f77ee2e976798af0be47c/src/ansys/pytwin/evaluate/twin_model.py)